api/utils/auth.py

```diff
@@ -0,0 +1,77 @@
+from fastapi import FastAPI, Depends, HTTPException, status
+from fastapi.security import OAuth2PasswordBearer, OAuth2PasswordRequestForm
+from sqlalchemy.orm import Session
+from pydantic import BaseModel
+import jwt
+from typing import Optional
+from datetime import datetime, timedelta
+from api.v1.models.user import User
+import os
+from jose import JWTError
+import bcrypt
+from api.v1.schemas.token import TokenData
+from api.db.database import get_db
+
+# Define your JWT secret and algorithm
+SECRET_KEY = os.getenv("SECRET_KEY", "MY SECRET KEY")
+ALGORITHM = os.getenv("ALGORITHM", "HS256")
```

Comment:
It's better to put all env variables in a single file, and import from there. There's a way to do that in FastApi, but you can just move all envs to a single file for now